client/src/elm/Translate.elm

```diff
@@ -413,6 +413,7 @@ type TranslationId
     | AILaboratoryTask AILaboratoryTask
     | All
     | AllowedValuesRangeHelper FloatInputConstraints
+    | AlmostEverydayOption
```

Comment:
Drop the `Option` suffix, here, and everywhere bellow.

General  note - if you decide on standard, it should be implemented everywhere.
So, if you decided to have `Option` suffix, there should not be sets without it.